chef-config/lib/chef-config/config.rb

```diff
@@ -690,6 +690,11 @@ def self.windows_home_path
       default :watchdog_timeout, 2 * (60 * 60) # 2 hours
     end
 
+    # Only use fips compliant algorithms
+    default( :fips_mode ) do
```

Comment:
And mostly where I see this being used is in remote_file fallback code.  If we find an MD5 checksum in the cache control data, but we have openssl_fips_mode set, then you're getting a new download.  It won't actually be used there to switch people to using SHA256, since everyone will be upgraded to SHA256, but it will indicate the capabilities of the openssl library.

Possibly this shouldn't even be a configurable...